<launch>
    <arg name="camera" value="/camera/color/image_raw" />

    <node name="lane_detection" pkg="turtlebot3_lane_detection" type="lane_detection" output="screen">
        <param name="camera_topic" value="$(arg camera)"/>
    </node>

</launch>